documentation/modules/rn-28-resolved-issues.adoc

```diff
@@ -7,6 +7,33 @@
 
 {project-first} 2.8 has the following resolved issues:
 
+[id="resolved-issues-2-8-1_{context}"]
+== Resolved issues 2.8.1
+
+.VMs with mutually exclsuive spec values as source VM and destination VM cannot be migrated
+In earlier releases of {project-short}, it was possible for a VM to have the `spec` value `Running` as a source VM with the `spec` value `runStrategy` as a destination VM. This caused migrations to fail because `Running` and `runStrategy` are mutually exclusive. This issue has been resolved in {project-short} 2.8.1. link:https://issues.redhat.com/browse/MTV-2092[(MTV-2092)]
+
+.The code base of MTV contains a deprecated value for a parameter
+In earlier releases of {project-short}, the code base contained the deprecated value `vm.Spec.Running`, which caused some migrations to fail. This issue has been resolved in {project-short} 2.8.1. link:https://issues.redhat.com/browse/MTV-2105[(MTV-2105)]
+
+.Migrations using custom templates fail if a user-supplied name is too long
+In earlier releases of {project-short}, migrations that used custom templates failed if a user-supplied was too long because it exceeded 63 characters. This issue has been resolved in {project-short} 2.8.1. Now {project-short} automatically truncates names that are too long. link:https://issues.redhat.com/browse/MTV-2244[(MTV-2244)]
+
+.User supplied template names are not validated against the RFC DNS1123 standard
+In earlier releases of {project-short}, user-supplied template names were not validated against the RFC DNS1123 standard. This caused migrations of VMs with template names that did not conform to RFC DNS1132 to fail. This issue has been resolved in {project-short} 2.8.1. link:https://issues.redhat.com/browse/MTV-2249[(MTV-2249)]
+
+.Disk names are changed after migration
+In earlier releases of {project-short}, migrating a VM that had disks with different buses resulted in disk names being changed after migration. This issue has been resolved in {project-short} 2.8.1. link:https://issues.redhat.com/browse/MTV-2266[(MTV-2266)]
```

Comment:
Should be https://issues.redhat.com/browse/MTV-2262 but we found issues with the fix and moving it out from 2.8.1. Please remove this part